src/service/url-replacements-impl.js

```diff
@@ -1049,7 +1049,9 @@ export class UrlReplacements {
     const url = element.getAttribute('src');
     const vars = Object.create(null);
     this.expandStringSync(url, /* opt_bindings */ undefined, vars);
-    const varNames = Object.keys(vars);
+    const varNames = Object.keys(vars)
+        // Strip arguments from collect_vars to match whitelist api.
+        .map(key => key.substring(0, key.indexOf('(')));
```

Comment:
Seems a bit hacky?

Reply:
I agree. A larger refactor is needed to be less so. Wasn't sure if we should patch and clean up after.

Reply:
indeed. is it possible to have spaces before `(`?

chatted offline, a better fix would redefine this `collectUnwhitelistedVarsSync` method, and move whitelist logic to the caller. 

I'm fine with a quick fix like this to patch prod.

Reply:
Should not be possible to have spaces before arguments.

Reply:
I thought we allow `/path/CLIENT_ID (xxx)` ? or did it get normalized somewhere down the path?

Reply:
We construct these in `maybeCollectVars_` with the macro name coming from the macro definition, not the url.

Reply:
oh, that's good.